public/app/frontend/src/components/link/link.html.twig

```diff
@@ -18,13 +18,6 @@ example:
     }%}
 #}
 <a class="link{{ newTab ? " link--new-tab" : "" }}"
-   href="{{ url }}" {{ newTab ? " target='_blank' rel='noreferrer noopener'" : "" }} {{ onClick ? " onclick='" ~ onClick ~ "'" : "" }}>
-    {{ label }}
-    {% if newTab and newTabVisuallyHidden %}
-        <span class="visually-hidden">
-        (opens in a new tab)
-    </span>
-    {% elseif newTab %}
-        (opens in a new tab)
-    {% endif %}
+   href="{{ url }}" {{ newTab ? " target='_blank' rel='noreferrer noopener'" : "" }} {{ onClick ? " onclick='" ~ onClick ~ "'" : "" }}>{{ label }}{% if newTab and newTabVisuallyHidden %}
```

Comment:
Agree, this one's a bit annoying because it needs to take a lot of optional params. I can a least split it over a few lines to make it easier to read though :)